lib/services/log.js

```diff
@@ -1,19 +1,41 @@
 'use strict';
 
-const _ = require('lodash'),
+var _ = require('lodash'),
   chalk = require('chalk'),
   path = require('path'),
   winston = require('winston'),
   util = require('util'),
   logLevel = process.env.LOG || 'info';
 
+/**
+ * add ELK logging
+ * note: this is in a function so we can test it
+ * @param {string} elk
+ */
+function addELK(elk) {
+  // write to elk & console if we have the CLAY_ELK env variable
+  if (elk) {
+    let logstashConf = {
+      host: elk.split(':')[0],
+      port: elk.split(':')[1],
+      node_name: 'clay'
+    };
+
+    require('winston-logstash'); // only require if we're using it
+
+    winston.add(winston.transports.Logstash, logstashConf);
+  }
+}
+
 // default to console logger, but pretty-print it
 winston.remove(winston.transports.Console);
 winston.add(winston.transports.Console, {
   colorize: true
 });
 winston.default.transports.console.level = logLevel;
 
+addELK(process.env.CLAY_ELK); // CLAY_ELK=domain.com:9200
```

Comment:
eh, everyone seems to refer to it as ELK still. We can definitely change this in the future. As for prefixing, I want to because hypothetically there could be non-clay logs being generated on the same server and this avoids any collision. (This env variable _specifically_ refers to _Clay_ logs)